app/login/index.js

```diff
@@ -31,12 +33,23 @@ exports.loginService = function loginService(parentWindow, callback) {
 	win.loadURL(`file://${__dirname}/login.html`);
 };
 
-exports.handleLoginDialogTry = function handleLoginDialogTry(window) {
+exports.handleLoginDialogTry = function handleLoginDialogTry(window, {ssoUser, ssoPasswordCommand}) {
+	console.log(ssoUser, ssoPasswordCommand);
 	window.webContents.on('login', (event, request, authInfo, callback) => {
 		event.preventDefault();
 		if (isFirstLoginTry) {
 			isFirstLoginTry = false;
-			this.loginService(window, callback);
+			if (ssoUser && ssoPasswordCommand) {
```

Comment:
Maybe some better error handling here, or at least some logs if the user provided one key but not the other...